### PR TITLE
chore(components): prep modal for retheme

### PR DESCRIPTION
**Pull request by darryltec**
## Motivations

This allows us to split off the styling of the modal when `jobber_retheme` class names exists on the body

### Added

- styling that only exists for `jobber_retheme`

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

Add the `jobber-retheme` class name on the body within the preview iframe

image

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Review by chris-at-jobber — COMMENTED** [review]
I think we want to fix this spacing to be even on left/right/bottom
image

File: packages/components/src/Modal/Modal.css [hunk 1]
```diff
@@ -64,6 +76,23 @@
   background-color: var(--color-surface--background);
 }
 
+:global(.jobber-retheme) .header {
+  background-color: transparent;
+}
+
+:global(.jobber-retheme) .header h3 {
+  font-size: var(--typography--fontSize-largest);
```
Comment (chris-at-jobber) [inline packages/components/src/Modal/Modal.css:84]:
```suggestion
  font-size: var(--typography--fontSize-largest);
  font-family: var(--typography--fontFamily-normal);
```

**Comment by darryltec** [comment]
@chris-at-jobber weird that you're seeing that. it should've gotten it as they're using the same variables 
image

**Review by chris-at-jobber — APPROVED** [review]
LGTM

**Comment by chris-at-jobber** [comment]
image

👌

**Pull request merged**